test/performance/odp_l2fwd.c

```diff
@@ -1243,10 +1494,17 @@ static void usage(char *progname)
 	       "  -p, --packet_copy       0: Don't copy packet (default)\n"
 	       "                          1: Create and send copy of the received packet.\n"
 	       "                             Free the original packet.\n"
-	       "  -y, --pool_per_if       0: Share a single pool between all interfaces (default)\n"
+	       "  -y, --pool_per_if       Create a packet (and packet vector) pool per interface.\n"
+	       "                          0: Share a single pool between all interfaces (default)\n"
 	       "                          1: Create a pool per interface\n"
 	       "  -n, --num_pkt <num>     Number of packets per pool. Default is 16k or\n"
 	       "                          the maximum capability. Use 0 for the default.\n"
+	       "  -u, --vector_mode       Enable vector mode.\n"
+	       "                          Supported only with scheduler packet input modes (1-3).\n"
+	       "  -w, --num_vec <num>     Number of vectors per pool.\n"
+	       "                          Default is num_pkts divided by vec_size.\n"
+	       "  -x, --vec_size <num>    Vector size. Default is max packet burst size.\n"
+	       "  -z, --vec_tmo_ns <ns>   Vector timeout in ns. Default timeout is 1 ms.\n"
```

Comment:
Please tweak the format a bit and read the default values from the defines, so they stay up to date.
E.g.
```
	       "  -x, --vec_size <num>    Vector size (default %i).\n"
	       "  -z, --vec_tmo_ns <ns>   Vector timeout in ns (default %llu ns).\n"
```

Otherwise the PR is looking good. You can add my reviewed-by tags in the next revision. Also, please add Marvell's copyright to the header.

Reply:
Sure, thanks.